src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1297,13 +1297,13 @@ void OrbitMainWindow::OnTimerSelectionChanged(const orbit_client_protos::TimerIn
   bool is_live_function_data_view_initialized = live_functions_data_view != nullptr;
   if (timer_info) {
     ORBIT_CHECK(is_live_function_data_view_initialized);
-    const uint64_t function_id = timer_info->function_id();
+    const uint64_t function_id = app_->ProvideId(*timer_info);
     selected_row = live_functions_data_view->GetRowFromFunctionId(function_id);
     live_functions_data_view->UpdateSelectedFunctionId();
```

Comment:
So this only affects the histogram, but not the list in the Live tab, right?

Reply:
The only way it effects the list: it deselects the selected function upon a click on a manually instrumented timer. In such a case `selected_row` is set to `std::nullopt`.